modules/permutiveRtdProvider.js

```diff
@@ -260,6 +268,39 @@ function readSegments (key) {
   }
 }
 
+/**
+ * Bidder-specific extension data for use in the ORTB2 object.
+ */
+const extensionDataByBidder = {
+  ix: {
```

Comment:
Is `ix` the code for Index? or an alias we are using?

Reply:
It's the code Prebid specifies for Index: https://docs.prebid.org/dev-docs/bidders/ix.html